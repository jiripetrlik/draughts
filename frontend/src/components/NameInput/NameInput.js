import React from 'react';
import TextField from '@material-ui/core/TextField';

function NameInput() {
    return (
        <TextField id="name-input" label="Your name"/>
    );
}

export default NameInput
