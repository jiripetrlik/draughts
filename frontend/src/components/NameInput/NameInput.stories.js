import React from 'react'
import NameInput from './NameInput'

export default {
  component: NameInput,
  title: 'NameInput',
};

const Template = args => <NameInput/>;

export const Default = Template.bind({});
