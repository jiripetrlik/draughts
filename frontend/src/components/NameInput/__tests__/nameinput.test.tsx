import React from 'react';
import renderer from 'react-test-renderer';
import NameInput from '../NameInput'

test('snapshot test', () => {
    const tree = renderer
        .create(<NameInput />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
