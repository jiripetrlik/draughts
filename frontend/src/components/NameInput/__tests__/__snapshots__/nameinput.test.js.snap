// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
  onKeyDown={[Function]}
>
  <label
    className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
    data-shrink={false}
    htmlFor="name-input"
    id="name-input-label"
  >
    Your name
  </label>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      id="name-input"
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;
